compiler-rt/lib/scudo/standalone/CMakeLists.txt

```diff
@@ -113,6 +113,7 @@ set(SCUDO_SOURCES
   mem_map_linux.cpp
   release.cpp
   report.cpp
+  report_linux.cpp
```

Comment:
Should the corresponding `.h` file be listed too in the section above?

Reply:
Yep, thanks for spotting that. I've fixed it now.